build.sbt

```diff
@@ -114,7 +114,10 @@ lazy val D = new {
   val tapirPrometheusMetrics: ModuleID = "com.softwaremill.sttp.tapir" %% "tapir-prometheus-metrics" % V.tapir
   val micrometer: ModuleID = "io.micrometer" % "micrometer-registry-prometheus" % V.micrometer
   val micrometerPrometheusRegistry = "io.micrometer" % "micrometer-core" % V.micrometer
-  val scalaUri = "io.lemonlabs" %% "scala-uri" % V.scalaUri
+  val scalaUri = Seq(
```

Comment:
@bvoiturier, I think we need to find a replacement for this library.
It's not actively maintained and might cause conflicts as we upgrade other libraries.
https://github.com/lemonlabsuk/scala-uri